activities/ColorMyWorld.activity/js/activity.js

```diff
@@ -83,5 +83,19 @@ define([
 		var modeButton = document.getElementById("mode-button");
 		modepalette = new modepalette.ModePalette(modeButton, undefined);
 		colormyworld.change_areaCB(true,'World');
+	
+		// Full screen
+		document.getElementById("fullscreen-button").addEventListener('click', function() {
+			document.getElementById("main-toolbar").style.opacity = 0;
+			document.getElementById("canvas").style.top = "0px";
+			document.getElementById("unfullscreen-button").style.visibility = "visible";
+			resizeCanvas();
```

Comment:
Error in console for this line.
![Screenshot 2019-12-07 at 08 23 02](https://user-images.githubusercontent.com/8216039/70371424-ccffd480-18ca-11ea-9987-d183d338ee0f.png)